sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/models/MapTileset.java

```diff
@@ -1,450 +1,136 @@
-        return this.minZoom;
+        return minZoom;
     }
 
-    /**
-     * Set the minZoom property: The minimum zoom level.
-     *
-     * @param minZoom the minZoom value to set.
-     * @return the MapTileset object itself.
-     */
-    public MapTileset setMinZoom(Integer minZoom) {
-        this.minZoom = minZoom;
-        return this;
-    }
-
-    /**
-     * Get the maxZoom property: The maximum zoom level.
-     *
-     * @return the maxZoom value.
-     */
     public Integer getMaxZoom() {
-        return this.maxZoom;
+        return maxZoom;
     }
 
-    /**
-     * Set the maxZoom property: The maximum zoom level.
-     *
-     * @param maxZoom the maxZoom value to set.
-     * @return the MapTileset object itself.
-     */
-    public MapTileset setMaxZoom(Integer maxZoom) {
-        this.maxZoom = maxZoom;
-        return this;
+    public BoundingBox getBounds() {
+        return bounds;
     }
 
-    /**
-     * Get the bounds property: The maximum extent of available map tiles. Bounds MUST define an area covered by all
-     * zoom levels. The bounds are represented in WGS:84 latitude and longitude values, in the order left, bottom,
-     * right, top. Values may be integers or floating point numbers.
-     *
-     * @return the bounds value.
-     */
-    public List<Float> getBounds() {
-        return this.bounds;
-    }
+    public Center getCenter() {
+        return center;
+    }   
 
-    /**
-     * Set the bounds property: The maximum extent of available map tiles. Bounds MUST define an area covered by all
-     * zoom levels. The bounds are represented in WGS:84 latitude and longitude values, in the order left, bottom,
-     * right, top. Values may be integers or floating point numbers.
-     *
-     * @param bounds the bounds value to set.
-     * @return the MapTileset object itself.
-     */
-    public MapTileset setBounds(List<Float> bounds) {
+    private void setBounds(BoundingBox bounds) {
         this.bounds = bounds;
-        return this;
-    }
-
-    /**
-     * Get the center property: The default location of the tileset in the form [longitude, latitude, zoom]. The zoom
-     * level MUST be between minzoom and maxzoom. Implementations can use this value to set the default location.
-     *
-     * @return the center value.
-     */
-    public List<Float> getCenter() {
-        return this.center;
     }
 
-    /**
-     * Set the center property: The default location of the tileset in the form [longitude, latitude, zoom]. The zoom
-     * level MUST be between minzoom and maxzoom. Implementations can use this value to set the default location.
-     *
-     * @param center the center value to set.
-     * @return the MapTileset object itself.
-     */
-    public MapTileset setCenter(List<Float> center) {
+    private void setCenter(Center center) {
         this.center = center;
-        return this;
     }
-}
+    
+    private void setToMapTileset(MapTilesetPrivate mapTilesetPrivate) {
```

Comment:
setFromMapTilesetPrivate() should be the name. 

You are setting properties on a public MapTileset FROM a private MapTilesetPrivate